airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/BasicAcceptanceTests.java

```diff
@@ -934,7 +934,7 @@ void testSyncAfterUpgradeToPerStreamStateWithNoNewData(final TestInfo testInfo)
   }
 
   @Test
-  void testResetAllWhenSchemaIsModified() throws Exception {
+  void testResetAllWhenSchemaIsModifiedForLegacySource() throws Exception {
```

Comment:
I had to update this test to explicitly use a legacy postgres source that does not support per-stream, since that is the scenario in which all streams are reset as a result of a schema modification